extensions/types/vscode-mssql.d.ts

```diff
@@ -718,6 +731,34 @@ declare module 'vscode-mssql' {
 		isCrossPlatformCompatible: boolean;
 	}
 
+	export interface GetProjectPropertiesResult extends ResultStatus {
```

Comment:
too bad we need to redefine these interfaces.  I guess this is to support both vscode & ads with different definitions.

Reply:
Yeahhh.  Merging the two service interfaces would save a good bit of manual effort.